features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/utils/DrawableUtils.kt

```diff
@@ -31,4 +32,46 @@ internal class DrawableUtils(
         drawable.draw(canvas)
         return bitmap
     }
+
+    internal fun downscaleBitmapToApproxSize(
```

Comment:
perhaps this should be moved to a new BitmapUtils class, because it doesn't actually involve drawables

Reply:
In the end not necessary, since refactoring to perform the downscaling when the bitmap is first created removes this method